src/gateio_new_coins_announcements_bot/rotating_proxy.py

```diff
@@ -23,7 +23,6 @@ def init_proxy():
 
 
 def _fetch_proxies():
-    logger.info("Fetching proxies...")
```

Comment:
I was not referring to  this line but the one after this.
`_proxy_list`

This print/log statement is perfectly fine here, that way we know when we are polling new proxies.

Reply:
- [ ] Please remove `_proxy_list` and revert the logger.info call

Reply:
Ops, my bad